web_widget_char_switchcase/static/test/web_widget_char_switchcase.js

```diff
@@ -3,88 +3,87 @@ odoo.define_section('web_widget_char_switchcase', ['web.form_common', 'web.core'
 
     function createField(form_common, web_form_widgets, node) {
         var field_manager = new form_common.DefaultFieldManager(null, {});
-        var field = new web_form_widgets.FieldChar(field_manager, node);
-        field.$input = $('<input/>');
-        field.initialize_content();
-        return field;
+        var fieldWidget = new web_form_widgets.FieldChar(field_manager, node);
+        fieldWidget.initialize_content();
+        return fieldWidget;
     }
 
     test('Default does nothing', function(assert, form_common, core, web_form_widgets) {
-        this.field = createField(form_common, web_form_widgets, {'attrs': {}});
+        var fieldWidget = createField(form_common, web_form_widgets, {'attrs': {}});
 
         var orig_val = 'Hello World!';
-        this.field.$input.val(orig_val);
-        this.field.$input.keyup();
-        assert.strictEqual(this.field.$input.val(), orig_val);
+        fieldWidget.$input.val(orig_val);
+        fieldWidget.$input.keyup();
```

Comment:
fieldWidget.$input.trigger('keyup')

Reply:
Done in https://github.com/OCA/web/pull/1079/commits/63e2623b69d22a27b224fe80586e2fef4a9ac99e